src/main/scala/firrtl/AddDescriptionNodes.scala

```diff
@@ -31,33 +31,33 @@ private case object EmptyDescription extends Description {
 
 private case class DescribedStmt(description: Description, stmt: Statement) extends Statement with HasDescription {
   def serialize: String = s"${description.serialize}\n${stmt.serialize}"
-  def mapStmt(f: Statement => Statement): Statement = f(stmt)
-  def mapExpr(f: Expression => Expression): Statement = this.copy(stmt = stmt.mapExpr(f))
-  def mapType(f: Type => Type): Statement = this.copy(stmt = stmt.mapType(f))
-  def mapString(f: String => String): Statement = this.copy(stmt = stmt.mapString(f))
-  def mapInfo(f: Info => Info): Statement = this.copy(stmt = stmt.mapInfo(f))
-  def foreachStmt(f: Statement => Unit): Unit = f(stmt)
-  def foreachExpr(f: Expression => Unit): Unit = stmt.foreachExpr(f)
-  def foreachType(f: Type => Unit): Unit = stmt.foreachType(f)
-  def foreachString(f: String => Unit): Unit = stmt.foreachString(f)
-  def foreachInfo(f: Info => Unit): Unit = stmt.foreachInfo(f)
+  def mapStmt(f:       Statement => Statement):   Statement = f(stmt)
```

Comment:
This alignment seems a bit overzealous.

Reply:
I think in this case it is a bit, but there are other cases where I think the alignment aids readability.
How overzealous do you think it is.
And would you like to join the formatting meeting at 1:30 on Friday to weigh in?